reporter/otlp_reporter.go

```diff
@@ -728,10 +731,32 @@ func addProfileAttributes(profile *profiles.Profile,
 	indices := make([]uint64, 0, len(attributes))
 
 	addAttr := func(attr attrKeyValue) {
-		if attr.value == "" {
+		var attributeCompositeKey string
+		var attributeValue common.AnyValue
+
+		switch attr.value.(type) {
+		case string, int64:
+			//Supported types, continue to handling below
+		default:
 			return
```

Comment:
This will change the signature of `addProfileAttributes()` but an error should be returned here. As `value` did change to `any` in `attrKeyValue` it now can be of any type.